composer.lock

```diff
@@ -0,0 +1,3401 @@
+{
```

Comment:
I don't see the value of adding the `composer.lock` file to a package, as Composer ignores it by default.